components/prism-arturo.js

```diff
@@ -0,0 +1,72 @@
+Prism.languages.arturo = {
+	'comment': {
+		pattern: /;.*/,
+		greedy: true
+	},
+
+	'character': {
+		pattern: /`.`/,
+		alias: 'char',
+		greedy: true
+	},
+
+	'string': {
+		pattern: /"(?:[^"\\\r\n]|\\.)*"/,
+		greedy: true
+	},
+
+	'regex': {
+		pattern: /\{\/.*?\/\}/,
+		greedy: true
+	},
+
+	'multistring': {
+		pattern: /"..."|».*|\{:[\s\S]*?:\}|\{[\s\S]*?\}|^-{6}$[\s\S]*/m,
```

Comment:
Sorry, with `"..."` I meant the single-line strings. I only used three dots because I wasn't sure about the correct pattern at the time.
```suggestion
		pattern: /».*|\{:[\s\S]*?:\}|\{[\s\S]*?\}|^-{6}$[\s\S]*/m,
```

Reply:
Haha. I have to admit it totally confused me. And the absurd reason is: Arturo *did* use `...` as string-block delimiters at some point in the past and I was left like "ok, I forgot about this... I thought I had removed it!" lol

Reply:
lol That the first time I guess syntax correctly.